app/helpers/users_helper.rb

```diff
@@ -0,0 +1,12 @@
+module UsersHelper
+  def gravatar_url(user)
+    hash = Digest::MD5.hexdigest(user.email)
+    "https://www.gravatar.com/avatar/#{hash}"
+  end
+
+  def has_gravatar?(user)
+    gravatar_url = gravatar_url(user)
+    response = HTTParty.get(gravatar_url, query: {d: '404'}).code
+    return response != 404
+  end
```

Comment:
That said, I like adding `gravatar_url` in `UsersHelper` more than installing another gem for this.